jest.config.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
       "<rootDir>/config/jest/fileTransform.js",
   },
   transformIgnorePatterns: [
-    "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$",
     "^.+\\.module\\.(css|sass|scss)$",
+    "<rootDir>/node_modules/(?!react-dnd|dnd-core|@react-dnd|react-dnd-scrolling|react-dnd-html5-backend)",
```

Comment:
on node modules, i added the `[/\\\\]` on purpose. 

[/\\\\]node_modules[/\\\\]: This part matches the string "/node_modules/" in the file path. The [/\\\\] part matches either a forward slash (/) or a backslash (\), so this works on both Unix-like systems (which use forward slashes in file paths) and Windows (which uses backslashes). The double backslashes are needed because backslashes are escape characters in regular expressions, so you need to use two backslashes to represent a single backslash.